src/main/java/cz/brmlab/yodaqa/flow/dashboard/snippet/SnippetIDGenerator.java

```diff
@@ -0,0 +1,18 @@
+package cz.brmlab.yodaqa.flow.dashboard.snippet;
+
+
+/**
+ * This class should generate unique ID's for Snippets
```

Comment:
The first sentence in javadoc is special, I think.  We should be careful to end it with a fullstop.